src/Fetcher/ConfTechFetcher.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+
+namespace App\Fetcher;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
```

Comment:
You call classes from Guzzle, but it does not seem installed. May be you should replace `php-http/curl-client` by https://github.com/php-http/guzzle6-adapter